test/grouping.jl

```diff
@@ -1320,13 +1320,17 @@ end
 
     df = DataFrame(a=[1, 1, 2, 2, 2], b=1:5)
     gd = groupby_checked(df, :a)
-    @test_throws ArgumentError combine(gd)
+    @test size(combine(gd)) == (0, 1)
+    @test names(combine(gd)) == ["a"]
 end
 
 @testset "GroupedDataFrame dictionary interface" begin
     df = DataFrame(a = repeat([:A, :B, missing], outer=4), b = repeat(1:2, inner=6), c = 1:12)
     gd = groupby_checked(df, [:a, :b])
 
+    @test gd[1] == DataFrame(a=[:A, :A], b=[1, 1], c=[1, 4])
+    @test_throws ArgumentError gd[1, 1]
```

Comment:
These are not really the dictionary interface, right? Don't we have another set of tests for integer indexing?

Reply:
This is testing for a bug introduced by dictionary interface addition.
Also essentially the error that user is likely to do is write `gd[1,1]` instead of `gd[(1,1)]`. But I will have a look if there is some better place for it.